docs/user-guide/11-target-specifics.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: user-guide
+permalink: /user-guide/target-specifics
+---
+
+Target specific functionalities
+============================
+
```

Comment:
We likely should add a section on "compiling to specific spirv versions".

We should explain how capabilities can affect the compiled spirv version for `-target spirv` and `emit-spirv-via-glsl`:
`-capability spirv_1_5` targets 1.5, `-capability spirv_1_6` targets 1.6, etc

Reply:
Thanks for reminding me of that.
I think it makes sense to talk about the compatibility profiles that are specific to SPIR-V and others.
But I am little hesitant that it may blot the document with not much of actual information.

Reply:
@ArielG-NV , can you give me some ideas of what could be helpful to include in this document regarding the capability specifically for spirv target?

I feel like the ones you described seems too generic and obvious.
>`-capability spirv_1-5` compiles spirv 1.5.